io.github.fasthud.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.fasthud",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "fasthudgtk4devel",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-config/MangoHud:rw"
```

Comment:
why do you need this xdg config?

Reply:
To modify and create a backup of MangoHud.conf.

Reply:
why read write? Why shall this modify another app conf?